sites/geohub/src/components/util/stac/StacApiExplorer.svelte

```diff
@@ -291,6 +311,15 @@
 					map.setFeatureState(clickedFeatures[0], { click: true });
 					stacDatasetFeature = await getProductFeature(itemIds);
 				}
+				if (selectedTool) {
+					for (const f of clickedFeatures.slice(0, clickedFeatures.length - 1)) {
+						map.setFeatureState(f, { click: false });
+					}
+					clickedFeatures = [clickedFeatures.at(-1)];
+					const itemIds = clickedFeatures.map((f) => f.properties.id);
+					map.setFeatureState(clickedFeatures[0], { click: true });
+					stacDatasetFeature = await getToolsFeature(itemIds);
```

Comment:
@JinIgarashi  The stacDatasetFeature returned here doesn't seem to work with the MiniMap component to show a preview

Reply:
@Thuhaa If it is not work, could you fix it?